File: ApplicationServer/src/models/manhwa-list/manhwa-list.ts
import { User } from "../user/user";
import { ManhwaListEntry } from "./manhwa-list-entry";

export class ManhwaList {
  listEntries: ManhwaListEntry[];
  owner: User;

  constructor(
    owner: User,
    { listEntries }: { listEntries: ManhwaListEntry[] } = { listEntries: [] }
  ) {
    this.owner = owner;
    this.listEntries = listEntries;
  }
}

File: ApplicationServer/ManhwaTracker/ManhwaTrackerApplicationServer/Models/ManhwaList/ManhwaList.cs
using System.ComponentModel.DataAnnotations;
namespace ManhwaTrackerApplicationServer.Models.ManhwaList;

public class ManhwaList
{
    [Key]
    public int Id { get; set; }
    public IEnumerable<ManhwaListEntry> ListEntries { get; set; }
    public User.User Owner { get; set; }
}